src/cfnlint/data/CloudSpecs/us-east-1.json

```diff
@@ -102604,12 +102693,6 @@
           "Required": false,
           "UpdateType": "Immutable"
         },
-        "PrivateIpAddress": {
```

Comment:
Makes me a little nervous but the current documentation says its currently not supported.